LoopKitUI/Views/Information Screens/CorrectionRangeInformationView.swift

```diff
@@ -9,13 +9,14 @@
 import SwiftUI
 
 public struct CorrectionRangeInformationView: View {
-    let blueGray = Color("Blue Gray", bundle: Bundle(for: DismissibleHostingController.self))
-    var exitPage: (() -> Void)
+    var onExit: (() -> Void)?
     var mode: PresentationMode
     
-    public init(exitPage: @escaping (() -> Void),
+    @Environment(\.presentationMode) var presentationMode
```

Comment:
Remembering how the SwiftUI `presentationMode` is passed through th environment makes me think we should do the same with our own PresentationMode... except for the unfortunate naming collision, which might make it not worth it.

(Just thinking out loud here, no action needed)